cfgov/cfgov/urls.py

```diff
@@ -279,6 +280,13 @@
         name='cckbyo'),
     # Form csrf token provider for JS form submission
     url(r'^token-provider/', token_provider),
+
+    url(r'^platzhalter/$', TemplateView.as_view(
+        template_name='ccdb5_landing_page.html')),
+    url(r'^platzhalter/api/v1/_suggest', CCDB5.suggest),
+    url(r'^platzhalter/api/v1/export', CCDB5.export),
+    url(r'^platzhalter/api/v1/(?P<id>[0-9]+)$', CCDB5.document),
+    url(r'^platzhalter/api/v1/', CCDB5.search),
```

Comment:
I would recommend feature flagging the URLs along with the mega menu below.

I'd also recommend going ahead and using the existing CCDB landing page URL here, but because it's an `include()` (and our flagged urls can't fall back on includes yet) it'll be a little wonky:

```python

flagged_url('CCDB5_RELEASE ', 
            r'^data-research/consumer-complaints/',
            include_if_app_enabled('complaintdatabase', 'complaintdatabase.urls')),
            state=False,
            fallback=TemplateView.as_view(template_name='ccdb5_landing_page.html'))
```

Which will serve the include if the flag is `False` and will "fall back" to the `TemplateView` if it is `True`.

I am very close to merging #2854 which will provide a [much nicer pattern for feature flagging a set of URLs like the `api/` URLs](https://github.com/cfpb/cfgov-refresh/blob/490deb2a88fc15d2fff3781da31ae482927fd08d/cfgov/cfgov/urls.py#L406-L433).

```python
with flagged_urls(`CCDB5_RELEASE`) as url:
    ccdb5_patterns = [
        url(r'^platzhalter/api/v1/_suggest', CCDB5.suggest),
        url(r'^platzhalter/api/v1/export', CCDB5.export),
        url(r'^platzhalter/api/v1/(?P<id>[0-9]+)$', CCDB5.document),
        url(r'^platzhalter/api/v1/', CCDB5.search)
    ]
urlpatterns += ccdb5_patterns
```